backend/services/location.service.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable func-names */
+const { Location } = require('../models/dictionaries/location.model');
+const DatabaseError = require('../errors/database.error');
+
+const LocationService = {};
+
+function validateLocations(locationsarray){
+
+  if(!Array.isArray(locationsarray)){
+    throw "locations array must be of type array"
```

Comment:
please improve the message to "locations input must be of type array"

Reply:
message improved